bin/travis-ci/check_tag.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+# echo "Installing shtdlib"
+sudo curl -s -L -o /usr/local/bin/shtdlib.sh https://github.com/sdelements/shtdlib/raw/master/shtdlib.sh
+sudo chmod 775 /usr/local/bin/shtdlib.sh
+# shellcheck disable=SC1091
+source /usr/local/bin/shtdlib.sh
+color_echo green "shtdlib.sh installed successfully"
+
+# Get the latest tag from GitHub
+latest_tag=$(git tag -l | sort --version-sort | tail -n1)
+color_echo green "Latest Git tag: '${latest_tag}'"
+
+# Get the latest tag from the CHANGELOG
+changelog_ver=$(grep -oP '\[v\d\.\d\.\d\]' CHANGELOG.md | tr -d '[]' | sort -nr | head -n1)
+color_echo green "CHANGELOG version: '${changelog_ver}'"
+
+# Get iteration from DEB builder configuration
+iteration_ver=$(grep -A1 iteration dc.deb.yml | tail -n1 | cut -d'"' -f2)
```

Comment:
I think you can safely add double quotes around the whole thing because the other ones are inside $()

Reply:
Done